mud_manager.c

```diff
@@ -2586,6 +2699,7 @@ void initialize_MongoDB()
     policies_collection = mongoc_client_get_collection (client, mongoDb_name, mongoDb_policies_collection);
     mudfile_collection = mongoc_client_get_collection (client, mongoDb_name, mongoDb_mudfile_coll);
     macaddr_collection = mongoc_client_get_collection (client, mongoDb_name, mongoDb_macaddr_coll);
+    vlan_collection = mongoc_client_get_collection (client, mongoDb_name, mongoDb_vlan_coll);
```

Comment:
The mud_show_db command should be updated to dump the current state of the vlan_collection following the dump of the other collections.

Reply:
Right.

Reply:
Actually, mud_show_db just dumps everything for all collections.  No change needed.

Reply:
Are you sure? I see mu_show_db dumping each collection ( mudfile, policies, macaddr) individually. E.g. 

echo
echo "MUD MACADDR COLLECTION"
echo

mongo mud_manager --eval "db.macaddr.find().forEach(printjson)"